cli/multipart_upload.go

```diff
@@ -0,0 +1,71 @@
+package cli
+
+import (
+	"github.com/minio/cli"
+	"github.com/minio/pkg/v3/console"
+	"github.com/minio/warp/pkg/bench"
+)
+
+var multipartUploadFlags = []cli.Flag{
+	cli.IntFlag{
+		Name:  "parts",
+		Value: 100,
+		Usage: "Number of parts to upload for each multipart upload",
+	},
+	cli.StringFlag{
+		Name:  "part.size",
+		Value: "5MiB",
+		Usage: "Size of each part. Can be a number or MiB/GiB. Must be a single value >= 5MiB",
+	},
+	cli.IntFlag{
+		Name:  "part.concurrent",
+		Value: 20,
+		Usage: "Run this many concurrent operations per each multipart upload. Must not exceed obj.size/part.size",
+	},
+}
+
+var MultiPartUploadCombinedFlags = combineFlags(globalFlags, ioFlags, multipartUploadFlags, genFlags, benchFlags, analyzeFlags)
+
+// MultipartUpload command
+var multipartUploadCmd = cli.Command{
+	Name:   "multipart-upload",
```

Comment:
Let's be a bit consistent in our naming.
```suggestion
	Name:   "multipart-put",
```